stable/polaris/templates/dashboard.rbac.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if and .Values.dashboard.enable .Values.rbac.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "polaris.fullname" . }}-dashboard
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "polaris.labels" . | nindent 4 }}
+---
```

Comment:
Helm best practices suggest that each resources should be it's own template.  That's up to you, just mentioning it.

https://helm.sh/docs/chart_best_practices/#structure-of-templates

Reply:
Hmm good to know. I'm going to leave this for now just to avoid breaking anything.